services/user.py

```diff
@@ -0,0 +1,49 @@
+from db.models import User
+
+
+def create_user(username: str,
```

Comment:
fix formatting

Reply:
and rewrite usages of User everywhere in according to checklis.md (get_user_model())